docs/faq.asciidoc

```diff
@@ -163,3 +163,31 @@ String encoded = ...;
 int scale = ...;
 final BigDecimal decoded = new BigDecimal(new BigInteger(Base64.getDecoder().decode(encoded)), scale);
 ----
+
+== How to change the offsets of the source database?
+
+[WARNING]
+This is a highly technical operation manipulating Kafka Connect internals.
+Please use this only as the last resort solution.
+
+Sometimes the database log contains an invalid data (like invalid date) that needs to be skipped or it is necessary to reprocess part of the log from the past.
+There is no straight way how to achieve this operation but there is a workaround that manipulates Kafka Connect's internal data.
```

Comment:
We actually have the "skip invalid events option" for MySQL. Can you reference it here?